lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -467,6 +467,7 @@ class IdentityCreatorController extends BaseController {
   void closeView(BuildContext context) {
     isCompressingInlineImage.value = false;
     clearFocusEditor(context);
+    publicAssetController?.discardChanges();
```

Comment:
Maybe instance of `publicAssetController` is not available anymore? Can we test it?

Reply:
Tested with identity creator controller